crates/storage/provider/src/providers/historical.rs

```diff
@@ -64,6 +64,24 @@ impl<'a, 'b, TX: DbTx<'a>> AccountProvider for HistoricalStateProviderRef<'a, 'b
             Ok(self.tx.get::<tables::PlainAccountState>(address)?)
         }
     }
+
+    /// Get account code by its address
```

Comment:
This should mention that this will return None if
the account does not exist
the account is not a contract